src/antsibull/build_ansible_commands.py

```diff
@@ -481,11 +460,6 @@ def compile_collection_exclude_paths(collection_names: Collection[str],
             for file in files:
                 all_files.append(os.path.normpath(os.path.join(directory, file)))
 
-        def ignore_file(prefix: str, filename: str):  # pylint: disable=unused-variable
-            if filename in all_files:
-                result.add(prefix + filename)
-                ignored_files.add(prefix + filename)
-
```

Comment:
I'd keep this unless we remove the compile_collection_exclude_paths function.

Reply:
This is a piece of code that was never used, and thought for further extensions of that function. Since we don't extend that function but remove it (once we drop support for Ansible < 8), I don't see a point in keeping this dead code.

Reply:
Fair enough. Thanks.

/resolve